src/Sulu/Bundle/MediaBundle/Media/FormatCache/LocalFormatCache.php

```diff
@@ -213,6 +211,6 @@ protected function getFileNameFromUrl($url)
             throw new ImageProxyInvalidUrl('No `filename` was found in the url');
         }
 
-        return $fileName;
+        return \urldecode($fileName);
```

Comment:
Looks like we are using `rawurlencode` here: https://github.com/sulu/sulu/blob/7b1490639e12d21d00a0a743b12f8aec833e84d9/src/Sulu/Bundle/MediaBundle/Media/FormatCache/LocalFormatCache.php#L119

Then we should also use `rawurldecode`:

```suggestion
        return \rawurldecode($fileName);
```